.gitignore

```
@@ -8,3 +8,4 @@
 __pycache__
 /cypress/videos/
 /cypress/screenshots/
+/.venv
```

Are you sure about the `/` here? Maybe it should be:

```suggestion
.venv
```

---

yes, it will only ignored in nrepo root then 😉